File: code_blocks/integrations/third-party-integrations/segment_1.dart
import 'package:purchases_flutter/purchases_flutter.dart';
import 'package:segment_flutter/segment_flutter.dart';

Future<void> setup() async {
  await Purchases.configure(
    PurchasesConfiguration('public_sdk_key')..appUserId = 'my_app_user_id',
  );
  final analytics = SegmentFlutter(writeKey: 'SEGMENT_WRITE_KEY');
  analytics.identify(userId: 'my_app_user_id');
}

File: code_blocks/integrations/third-party-integrations/segment_1.ts
import Purchases from '@revenuecat/purchases-react-native';
import { createClient } from '@segment/analytics-react-native';

async function setup() {
  await Purchases.configure({ apiKey: 'public_sdk_key', appUserID: 'my_app_user_id' });
  const segment = createClient({ writeKey: 'SEGMENT_WRITE_KEY' });
  segment.identify('my_app_user_id');
}

